package.json

```diff
@@ -15,8 +15,8 @@
     "backbone": "1.3.3",
     "backbone.paginator": "2.0.6",
     "bootstrap": "4.0.0-beta.2",
-    "coffee-loader": "0.7.3",
     "classnames": "2.2.5",
+    "coffee-loader": "0.7.3",
```

Comment:
Looks like when I ran `npm install @edx/studio-frontend@0.8.4 --save --save-exact` it alphabetized this package as well